src/events/messageCreate.js

```diff
@@ -0,0 +1,43 @@
+const youchatwrapper = require("@codernocook/youchatwrapper");
+
+module.exports = {
+  name: "messageCreate",
+  execute: async (message, client) => {
+    const replied =
+      message.reference !== null
+        ? await message.channel.messages.fetch(message.reference.messageId)
+        : null;
+
+    if (
+      message.content.includes(`<@${client.user.id}>`) ||
+      (replied !== null ? replied.author.id === client.user.id : 0)
+    ) {
+      message
```

Comment:
Errors thrown inside the `then` method can only be caught by the events error handler by using `await` since the `reply` method returns a `Promise`, that's why `execute` is an async function.
```suggestion
      await message
```